todo_app/app.py

```diff
@@ -83,11 +68,27 @@ def get_sort_parameters():
 @app.route('/', methods = ['GET'])
 def index():
     
+    member = get_member('alanjknight@hotmail.com')
```

Comment:
We should extract this member out to an environment variable, so it can be easily swapped in and out (without that I missed that this appeared twice, and was left confused trying to work out why I couldn't add cards!)
Having said that, all we use this member for is to get the first board associated with them. We could simplify this logic substantially by instead either:
* Using `<trello>/members/me/boards` to get the board ids which uses the member associated with the passed token, so we don't need to specify a member email address at all
* Or even simpler, just ask the user to specify a board id in the environment variable rather than an account, which allows for more control (it doesn't just use the first board) and saves us unnecessary calls to Trello to find out what other boards exist